content/tutorials/spatial_crs_coding/index.Rmd

```diff
@@ -31,46 +30,64 @@ options(stringsAsFactors = FALSE)
 
 Since long, coordinate reference systems in R (and many other tools) have been represented by so called 'PROJ.4 strings' (or 'proj4strings'), referring to the long-standing version 4 of the PROJ library.
-But, we will not use them here - it is counteradvised to use them any longer!
+But, we will not use them here!
+It is discouraged to use 'PROJ strings'[^projstring] any longer to represent a CRS; several string elements for CRSs are now deprecated or unsupported.
+Currently, PROJ (<https://proj.org>) regards PROJ strings only as a means of specifying a _coordinate operation_ (conversions or transformations _between_ CRSs).
+Performing coordinate operations is the main aim of PROJ.
 
-Let's just have **one last nostalgic peek** (and then, no more!!): the proj4string for the Belgian Lambert 72 CRS (EPSG:31370) "_was_":
+[^projstring]: 'PROJ string' is the term used in current PROJ documentation.
+Here, we only use 'PROJ.4 string' (or 'proj4string') when referring specifically to the PROJ string appearance in version 4 of the PROJ software.
+
+Let's just have **one last nostalgic peek** (and then, no more!!) to the proj4string for EPSG:31370, the Belgian Lambert 72 CRS [^belge72]:
 
     +proj=lcc +lat_1=51.16666723333333 +lat_2=49.8333339 +lat_0=90 +lon_0=4.367486666666666 +x_0=150000.013 +y_0=5400088.438 +ellps=intl +towgs84=-106.8686,52.2978,-103.7239,0.3366,-0.457,1.8422,-1.2747 +units=m +no_defs
 
-Several reasons, such as WGS84 getting outdated, higher accuracy requirements in transformations and the availability of better standards than proj4strings, have led to recent changes in PROJ and GDAL.
-This also implied reduced compatibility with proj4strings.
+[^belge72]: Note that the _currently returned_ PROJ string for EPSG:31370, if requested from PROJ ≥ 6 or GDAL 3 (not shown), lacks the datum reference which, in PROJ.4, was defined indirectly by `+towgs84` with the 7-parameter (Helmert) transformation to the WGS84 datum.
+Hence the current PROJ string is a deficient representation of EPSG:31370.
+
+Several reasons have led to recent changes in GDAL and PROJ, such as the use of WGS84 as a transformation hub while it is a dynamic CRS, higher accuracy requirements in transformations and the availability of better CRS specification standards than PROJ strings.
+The changes are included in **GDAL 3** and **PROJ ≥ 6**, which many R packages now support and promote.
+
+In consequence, support for PROJ strings to represent a CRS is reduced and discouraged.
+It _can_ still be done, preferrably adding the `+type=crs` element to distinguish such a string from modern PROJ strings.
+The latter represent a coordinate operation, not a CRS.
+Currently, support for most geodetic datums is already lacking in PROJ strings [^projcrs] (unless one defines it indirectly, but likely less accurately, with the now deprecated `+towgs84` key).
+The WGS84 ensemble datum [^wgs84] (EPSG:6326) is now by default assumed for a CRS declared with a PROJ string.
```

Comment:
```suggestion
The WGS84 ensemble datum [^wgs84] (EPSG:4326) is now by default assumed for a CRS declared with a PROJ string.
```

Reply:
The geodetic datum is [EPSG:6326](https://epsg.org/datum_6326/World-Geodetic-System-1984-ensemble.html). A 2D geographic CRS that uses this datum is [EPSG:4326](https://epsg.org/crs_4326/WGS-84.html), but there are others.

Reply:
Strange, I also looked this up, but google directed me to: [https://epsg.io/6326](https://epsg.io/6326) which apparently is not the same as the link you give (and which convinced me you made a typo). Confuses me 🤔

Reply:
Interesting and well seen @hansvancalster, thank you! The correct link at esg.io is https://epsg.io/6326-datum. It appears that the `AUTHORITY:CODE` implementation is software specific (in PROJ, it is limited to CRSs), and not a standard at all. The numeric EPSG codes can overlap between object types (such as a transformation and a datum, in this case).

```
$ projinfo EPSG:6326
input string: parsing of user string failed: crs not found
```

The best way to refer an EPSG object appears to be using a Uniform Resource Name (URN) - see https://docs.opengeospatial.org/pol/09-048r5.html.

```
$ projinfo urn:ogc:def:datum:EPSG::6326
WKT2:2019 string:
DATUM["World Geodetic System 1984",
    ELLIPSOID["WGS 84",6378137,298.257223563,
        LENGTHUNIT["metre",1]],
    ID["EPSG",6326]]
```

For clarity I'll replace with '[datum:EPSG::6326](https://epsg.org/datum_6326/World-Geodetic-System-1984-ensemble.html)' in the tutorial.

Reply:
Applied in 5de4e91e